### PR TITLE
Replace arrow functions for compatibility

### PR DESCRIPTION
**Pull request by idontbyte**
Replace arrow functions due to lack of support in IE / Safari leading to confused users in Getting Started document

### CODE REVIEW

File: src/tutorials/core/getting-started-quickstart/readme.md [hunk 1]
```diff
@@ -44,29 +44,29 @@ Create an _index.html_ file and add the following to it, making sure to point to
 This page consists of one text field ready for user input. Inside the `script` tag, add the following JavaScript to login to your deepstream server:
 
 ```javascript
-const client = deepstream('localhost:6020').login()
+const client = deepstream('localhost:6020').login();
```
Comment (yasserf) [inline src/tutorials/core/getting-started-quickstart/readme.md]: this should be var too then right?

**Comment by yasserf** [comment]
Hey @idontbyte,

This is quite interesting, we had all of our documentation in non es6 but started moving it across because of the whole transpiling craze that javascript documentation now has!

Shouldn't const be vars then? Also, can we not use semi columns? It doesn't affect code and we just removed them as well as part of our new style guide

Thanks!

**Comment by idontbyte** [comment]
Fine with constant being vars and no semi colons but a getting started guide shouldn't hinder the visitor using your platform. It sure had me a little frustrated as I hadn't encountered arrow functions before, I worked it out but many won't.

**Comment by idontbyte** [comment]
Updated request

**Comment by yasserf** [comment]
Perfect, thank you!

**Pull request merged**